tasks/patternpack.js

```diff
@@ -134,6 +139,24 @@ module.exports = function (grunt) {
     }
   }
 
+  function ensureFilesExist(options) {
+    var path = require("path");
+
+    // Create core css file if it does not exist
+    var coreCssPath = options.assets + "/" + options.css.preprocessor + "/" + options.css.fileName + "." + options.css.preprocessor;
```

Comment:
I think we should just create the "right" file extension when we have `sass` configured